src/components/App/App.js

```diff
@@ -0,0 +1,42 @@
+// import logo from './logo.svg';
+import './App.css';
+import movieData from '../../data';
+import AllMovies from '../AllMovies/AllMovies';
+import  { useState } from 'react';
+
+
+function App() {
+  const dummyMovies = [
```

Comment:
We used dummyMovies to test our App() function with a small set a data, to make sure we we're accessing the right data.